javascript/javascript1/week2/fullName.js

```diff
@@ -0,0 +1,30 @@
+//FLIGHT BOOKING FULLNAME FUNCTIONALITY
+
+console.log('\n***** Homework Javascript Week02 Task 01-FULL NAME" *****');
+
+
+function getFullname(firstname, surname, useFormalName,gender) {
+    if (firstname === '' && surname === '') {
+            return 'Please enter both First Name and Last Name';
+    } else if (useFormalName) {
+        if(gender=='male'){
```

Comment:
1. Add `toLowerCase()` just in case someone pushes the function with uppercase.
2. Use `===` as it's better practice

```suggestion
        if(gender.toLowerCase() === 'male') {
```

Reply:
DONE